Added s3 flag to delete audio file onto Amazon S3 post processing (#9)
awslabs/distill-cli: README.md

Distill CLI supports configurable language codes for transcription using Amazon Transcribe. Specify the language of the audio file with the `--language-code` or `-l` parameter (e.g., `--language-code es-US` for Spanish). If no language code is provided, it defaults to English (United States) (`en-US`). For a list of supported languages and their codes, see the [Amazon Transcribe Supported Languages Documentation](https://docs.aws.amazon.com/transcribe/latest/dg/supported-languages.html). Use `--help` for more information on available options. The prompt in the provided config.toml is adjusted so that the provided output matches the language in the transcript.

# S3 Object Deletion

> Distill CLI supports deletion of the audio recodings once transcribe and summarize activities performed with success. By using the `--delete-s3-object` or `-d` parameter (e.g., `--delete-s3-object Y`), the audio file previously uploaded onto Amazon S3 will be deleted. Use `--help` for more information on available options.

Reviewer: ```suggestion
Distill CLI supports the deletion of audio recordings after transcription and summarization activities have completed. Using the `--delete-s3-object` or `-d` parameter (e.g., `--delete-s3-object Y`), the audio file previously uploaded onto Amazon S3 will be deleted. Use `--help` for more information on available options.
```

Reviewer: In this pattern, would the user need to go into S3 to find the object number before deleting? And do you see someone running Distill after the fact to perform a deletion? 

It probably makes more sense to grab the S3 Object name after creation (see Line 164: `let s3_uri`), then after transcription and summarization have completed, to delete the object for the user. This keeps the full interaction at one command, and doesn't require someone to explicitly know the object ID.

Reviewer: Ugh, I think I misinterpreted how this is working, my apologies. So the way this works that is if you pass `Y` it will delete the resource when complete. However, it defaults to `N` which keeps the resource?

Reviewer 2: To confirm: Do we still require to implement your suggestion on grabbing the S3 Object name (or not)?

Reviewer: No, this is fine. Sorry for the confusion.
